css/css-logical/logical-longhand-properties.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8" />
+<title>Test for handling of logical and physical properties</title>
```

Comment:
Nit: Other tests in this suite have "CSS Logical Properties:" in the title. You might want to add it.
This test is about "4. Flow-Relative Box Model Properties" so I'd mention that in the tittle too.
What about something like: "CSS Logical Properties: Flow-Relative Box Model Properties".